pkg/get/tools.go

```diff
@@ -1609,5 +1609,39 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 		},
 	)
 
+	tools = append(tools,
+		Tool{
+			Owner:       "goreleaser",
+			Repo:        "goreleaser",
+			Name:        "goreleaser",
+			Version:     "0.177.0",
```

Comment:
Please remove the version so that it been be detected and let us know if that works as expected.

You may also be able to switch to Binary Template now

Reply:
fixed @alexellis, ready to merge, thank you